_client/src/Ui/Component/MainMenu.cpp

```diff
@@ -0,0 +1,50 @@
+#include "MainMenu.hpp"
+
+MainMenu::MainMenu() : QMenuBar()
+{
+    // File Menu
+    fileMenu->addAction(settingsAction);
+    connect(exitAction, &QAction::triggered, this, &MainMenu::close);
+    fileMenu->addAction(exitAction);
+
+    // Purchasing Module Menu
+    connect(purchaseOrdersAction, &QAction::triggered, this, &MainMenu::showPurchaseOrders);
+    purchasingMenu->addAction(purchaseOrdersAction);
+    purchasingMenu->addAction("Supplier Management");
```

Comment:
Don't use hardcoded strings

Use enum-map<enum, string> approach
Create `enum class ActionTypes` with action identificators and use `map<enum, string> ACTION_NAMES` to set names to the menu